src/Utils.js

```diff
@@ -213,6 +213,19 @@ export function coercePropValue(propName, propValue) {
     return propValue;
   }
 
+  // can be the empty string
```

Comment:
yep I know, but is it actually worth it to change it seeing as it'd be a such a minor, yet breaking, change?